docs/dev-process/coding-guidelines.md

```diff
@@ -0,0 +1,215 @@
+# C# Coding Guidelines
```

Comment:
This file looks like a copy pasted version of the one of PowerShell Core. Don't you think it makes sense to rather link it to it instead or highlight the differences if there are any? Just my 2 cents, you can still decide at the end.

Reply:
yah - referring to the PSCore doc makes sense. I'll do that